example_test.go

```diff
@@ -33,19 +33,6 @@ func ExampleNewBranca() {
 	fmt.Printf("Token: %s\n", token)
 }
 
-func ExampleBranca_SetTTL() {
-	key := "mysupppadupppasecretkeyforbranca"
-	brc, err := NewBranca([]byte(key))
-
-	if err != nil {
-		fmt.Printf("Error: %v\n", err)
-		return
-	}
-
-	// Set TTL for 1 minute
-	brc.SetTTL(60)
-}
-
 func ExampleBranca_IsExpired() {
```

Comment:
You have removed this method but kept the usage example. Please rewrite it with an actual usage example of `token.IsExpired`.

Reply:
Done, thank you for review.